test/controllers/thesis_controller_test.rb

```diff
@@ -4,7 +4,7 @@ class ThesisControllerTest < ActionDispatch::IntegrationTest
   # ~~~~~~~~~~~~~~~~~~~~~~~~~~ the submission system ~~~~~~~~~~~~~~~~~~~~~~~~~~
   test 'new prompts for login' do
     get '/thesis/new'
-    assert_response :redirect
+    assert_redirected_to '/users/auth/saml'
```

Comment:
Is this change related to anything else in the PR? I see that this matches what the test is for the transfer controller - so I'm reading this as "what the test should have been" rather than a substantive change. I'm fine with that, but don't want to miss anything.

Reply:
Yes, this change is unrelated to the goal of the PR. I noticed that the test doesn't really do what it seeks to, so I figured I'd fix it as I was writing similar tests for the TransferController.